aten/src/ATen/WrapDimUtils.h

```diff
@@ -27,6 +27,17 @@ static inline int64_t maybe_wrap_dim(int64_t dim, int64_t dim_post_expr, bool wr
   return dim;
 }
 
+static inline std::vector<bool> dim_list_to_vector(IntList dims, int64_t ndims, bool wrap_scalar=true) {
+  std::vector<bool> seen(ndims, false);
```

Comment:
I think it's ok to use `std::bitset<64>` here (with a check that `ndims < 64`). I don't think anyone is using 64-dim tensors, so let's avoid making everyone else pay the cost 😄